### DISCUSSION TITLE
UHCI is loaded despite being disabled with usb_uhci=false

### DISCUSSION THREAD

**Discussion by Vort** [bochs-emu/Bochs#117] [Q&A] [answered]
I was trying to make `sb16` work with Windows XP.
But when OS starts, such errors appears:
```text
00915731249i[UHCI  ] new IRQ line = 5
00915867260p[DEV   ] >>PANIC<< IRQ 5 conflict, SB16 with PIIX3 IRQ routing
```

Why it happens?
I have `usb_uhci=false` set in `plugin_ctrl:`, but still see this line:
`00000000000i[PLUGIN] init_dev of 'usb_uhci' plugin device by virtual method`
Is it possible to disable USB so it won't try to use IRQ 5 line?

**Comment by vruppert** [comment, marked as answer]
UHCI is a part of the i440FX chipset, so it's always loaded in this case. If you chose the i430FX chipset, there is no onboard UHCI support . Enabling UHCI in that case assigns it to a PCI slot.

**Reply by Vort** [reply]
Thanks.
Looks like changing chipset requires reinstalling Windows.
I've found better method to get rid of conflict - USB controller can be disabled in device manager.
